src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -14,13 +16,15 @@
     private final AuthenticationService authService;
     private final UserMapper userMapper;
 
-    public AuthenticationController(AuthenticationService authService, UserMapper userMapper) {
+    public AuthenticationController(AuthenticationService authService,
+                                    UserMapper userMapper) {
         this.authService = authService;
         this.userMapper = userMapper;
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
+    public UserResponseDto register(@RequestBody @Valid UserRequestDto requestDto)
+            throws NoSuchMethodException, MethodArgumentNotValidException {
```

Comment:
it seems to me that it looks strange when the controller, responsible for registration, throws NoSuchMethodException. This exception may need to be handled where it is thrown.